test/runtests.jl

```diff
@@ -10,8 +12,10 @@ using OMEinsum
     include("Transformations_test.jl")
     include("Site_test.jl")
     include("Quantum_test.jl")
-    include("Product_test.jl")
-    include("Chain_test.jl")
+    include("Lattice_test.jl")
+    include("Ansatz_test.jl")
+    # include("Product_test.jl")
+    # include("Chain_test.jl")
```

Comment:
```suggestion
    include("Product_test.jl")
    include("Chain_test.jl")
```
If we have `testskip` inside we don't need to comment out this, right?